### ISSUE TITLE
AutoMessages config probably needs to be updated..

### ISSUE THREAD

**Issue by BearNationSMP** [Pugabyte/BearNation#219]
Submitted by **Filid**

**Comment by FilidDamhsoir** [comment]
Message starts at line: 37
Issue: There is no member world anymore
Currently Enabled? Yes


Message starts at line: 48
Issue: Staff apps are always open.  May want to rewrite this one, and turn it on selectively when we really need staff.  Also maybe toggle it again soon as we have lost a few staff?
Currently Enabled? No

Message starts at line: 94
Issue: Guests can now claim plots in Creative
Currently Enabled? Yes

Message starts at line: 119
Issue: Follow us on twitter? We've tweeted twice in the last year, is this announcement still worthwhile? (and if so, then followup issue- our twitter game sucks) 
Currently Enabled? Yes

Message starts at line: 129
Issue: Stranded isn't new anymore. Should still be advertised probably, but maybe reword it?  Check out our pirate Adventure map Stranded (instructions/clicky thing)  
Currently Enabled? Yes

Message starts at line: 156
Issue:  /recipe is currently disabled
Currently Enabled? Yes

Message starts at line: Nowhere
Issue: We talked recently about better advertising the shub and the malls.  This is one opportunity to do so we are currently missing.  Same with Skyblock, pugmas/other events. It looks like the last event to be in here is the currently disabled message about ore hunt. 
Currently Enabled? Nope doesn't yet exist

I assume that the duplicate/similar messages is to boost signal on more important announcements like shops/mgn/map/votes/donations/discord
